migrations/20230714125055-create_stories_table.sql

```diff
@@ -0,0 +1,35 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS stories (
+    story_id SERIAL PRIMARY KEY,
+    author_id INT,
+    story_content TEXT,
+    created_at created_at_type,
+    deleted_at  deleted_at_type,
+    updated_at updated_at_type,
+    FOREIGN KEY (author_id) REFERENCES users(user_id)
+);
```

Comment:
A minor comment, because it mainly revolves around style, but having the FK constraint separately at the bottom somewhat breaks the flow of reading, when compared to having it inlined:

```suggestion
CREATE TABLE IF NOT EXISTS stories (
    story_id SERIAL PRIMARY KEY,
    author_id INT REFERENCES users(user_id),
    story_content TEXT,
    created_at created_at_type,
    deleted_at  deleted_at_type,
    updated_at updated_at_type
);
```

Having it inlined improves readability as the constraints for the column appear immediately at the same line.